src/commands/voucher.js

```diff
@@ -15,8 +16,11 @@ if (!fs.existsSync(vouchersFile)) {
 
 const ERROR_NOT_DM_MESSAGE = 'Carefull ! Yoou need to give your voucher code to the bot in a private message !'
 const INVALID_VOUCHER = 'Invalid voucher code'
+const VOUCHER_SUCCESS = 'WOW much money'
 
-function voucher (message, dogecoinNode, voucherCode) {
+const VOUCHER_AMOUNT = 10
+
+async function voucher (message, dogecoinNode, voucherCode) {
```

Comment:
idem dogecoinNode ne sert plus ? 
Ou alors il y a un truc que je n'ai pas saisi ^^

Reply:
Yeah same. I just forgot to commit.